metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -339,12 +413,23 @@
                         :transaction-id (long (:transaction_id result))})
           (su/query conn stmt)))))
 
+(defn- variable-association-concept?
```

Comment:
This check should be just checking the concept-type is `:variable-association`.

Reply:
Done.